SendWelcomeMessagesActivity/index.ts

```diff
@@ -116,20 +121,35 @@ Per approfondimenti ti invitiamo a consultare la sezione [servizi](ioit://SERVIC
     })
 ];
 
+const SERVICES_API_BASE_HEADERS = {
+  "Content-Type": "application/json"
+};
+
 /**
- * Send a single welcome message using the
- * Digital Citizenship Notification API (REST).
+ * Send a single welcome message using the Services API.
+ *
+ * If the apiKey is empty, the request will include the authorization
+ * headers that the Services API expects from the API Manager.
  */
 async function sendWelcomeMessage(
   url: string,
-  apiKey: string,
+  apiKey: Option<NonEmptyString>,
   newMessage: NewMessage
 ): Promise<number> {
+  const authHeaders = apiKey
+    .map<{ [key: string]: string }>(k => ({
+      "Ocp-Apim-Subscription-Key": k
+    }))
+    .getOrElse({
+      "x-subscription-id": "functions-app",
```

Comment:
I've a major concern here:  is this the expected pattern to call any API exposed by the functions ? if so, this should be put into some common shared module, ie. `io-functions-commons`, otherwise it would be hard to refactor and grasp since these fields are taken from API gateway _internals_ which may change sometime. Another option is to generate the client using io-utils to keep specs in sync.